docs/en/core-services/Caching.md

```diff
@@ -204,70 +190,90 @@ A typical configuration will have three tiers for Memory, SSD, and HDD.
 
-Alluxio uses evictors to decide which blocks to remove from Alluxio storage.
-Users can specify different Alluxio evictors to control the eviction process better.
+`The old eviction policies are now deprecated and Alluxio provided implementations are replaced with appropriate annotation policies.
+Alluxio supports emulation mode which annotates blocks based on custom evictor implementation. The emulation assumes the configured eviction policy creates
+an eviction plan based on some kind of order and works by regularly extracting this order to be used in annotation activities.`
 
-Out-of-the-box evictor implementations include:
+Out-of-the-box annotation implementations include:
 
-- **LRUEvictor**: Evicts the least-recently-used blocks until the required space is freed.
-**This is Alluxio's default evictor**.
-- **GreedyEvictor**: Evicts arbitrary blocks until the required space is freed.
-- **LRFUEvictor**: Evicts blocks based on least-recently-used and least-frequently-used with a
+- **LRUAnnotator**: Annotates the blocks based on least-recently-used order.
+**This is Alluxio's default annotator**.
+- **LRFUAnnotator**: Annotates the blocks based on least-recently-used and least-frequently-used orders with a
 configurable weight.
     - If the weight is completely biased toward least-recently-used, the behavior will be the same
-    as the LRUEvictor.
-    - The applicable configuration properties are `alluxio.worker.evictor.lrfu.step.factor` and
-    `alluxio.worker.evictor.lrfu.attenuation.factor` which can be found in
-    [configuration properties]({{ '/en/reference/Properties-List.html' | relativize_url }}#alluxio.worker.evictor.lrfu.step.factor)
-- **PartialLRUEvictor** : Evicts based on a least-recently-used policy but will choose one configured
-storage directory on the local worker with maximum free space and only evict from that directory.
-
-The evictor utilized by workers is determined by the Alluxio property
-[`alluxio.worker.evictor.class`]({{ '/en/reference/Properties-List.html' | relativize_url }}#alluxio.worker.evictor.class).
+    as the LRUAnnotator.
+    - The applicable configuration properties are `alluxio.worker.block.annotator.lrfu.step.factor` and
+    `alluxio.worker.block.annotator.lrfu.attenuation.factor`.
+
+The annotator utilized by workers is determined by the Alluxio property
+[`alluxio.worker.block.annotator.class`]({{ '/en/reference/Properties-List.html' | relativize_url }}#alluxio.worker.block.annotator.class).
 The property should specify the fully qualified class name within the configuration. Currently
 available options are:
 
-- `alluxio.worker.block.evictor.LRUEvictor`
-- `alluxio.worker.block.evictor.GreedyEvictor`
-- `alluxio.worker.block.evictor.LRFUEvictor`
-- `alluxio.worker.block.evictor.PartialLRUEvictor`
+- `alluxio.worker.block.annotator.LRUAnnotator`
+- `alluxio.worker.block.annotator.LRFUAnnotator`
```

Comment:
Could you add a section here for backward compatibility, ie. if users were using LRFU, what kind of error they might see and how to restore the previous behavior (ie. switch to LRFUAnnotator)?

Reply:
You mean putting this into a dedicated section?

> The old eviction policies are now deprecated and Alluxio provided implementations are replaced with appropriate annotation policies.
Alluxio supports emulation mode which annotates blocks based on custom evictor implementation. The emulation assumes the configured eviction policy creates
an eviction plan based on some kind of order and works by regularly extracting this order to be used in annotation activities.`

Reply:
Add to that the typical error message they will see if they forgot to change from the old property (ie. class not found) and what the corresponding new annotator classes are for the old evictors.

Reply:
ptal @ https://github.com/Alluxio/alluxio/pull/11700/commits/bf88b301150cce034a5e2765d944e661517d6554